app/dashboard/utils.py

```diff
@@ -203,7 +204,17 @@ def get_web3(network):
 
     """
     if network in ['mainnet', 'rinkeby', 'ropsten']:
-        return Web3(HTTPProvider(f'https://{network}.infura.io'))
+        if sockets:
+            provider = WebsocketProvider(f'wss://{network}.infura.io/ws')
+        else:
+            provider = HTTPProvider(f'https://{network}.infura.io')
+        w3 = Web3(provider)
+        if network == 'rinkeby':
+            w3.middleware_stack.inject(geth_poa_middleware, layer=0)
+        return w3
+    elif network == 'localhost' or 'custom network':
+        return Web3(Web3.HTTPProvider("http://testrpc:8545", request_kwargs={'timeout': 60}))
```

Comment:
Should we update this to use `settings.WEB3_HTTP_PROVIDER` so it'll be customizable in any case vs making this deterministic based on env?